src/software/ai/hl/stp/play/ball_placement/ball_placement_play_fsm.cpp

```diff
@@ -53,12 +126,12 @@ void BallPlacementPlayFSM::alignPlacement(const Update &event)
                 .normalize();
         Angle setup_angle = alignment_vector.orientation();
         setup_point       = event.common.world_ptr->ball().position() -
-                      2 * alignment_vector * ROBOT_MAX_RADIUS_METERS;
+                      2.5 * alignment_vector * ROBOT_MAX_RADIUS_METERS;
 
         align_placement_tactic->updateControlParams(
             setup_point, setup_angle, 0.0, TbotsProto::DribblerMode::OFF,
             TbotsProto::BallCollisionType::AVOID, {AutoChipOrKickMode::OFF, 0},
-            TbotsProto::MaxAllowedSpeedMode::PHYSICAL_LIMIT,
+            TbotsProto::MaxAllowedSpeedMode::STOP_COMMAND,
```

Comment:
why does alignment need to be slow? we're not dribbling at this point

Reply:
After we pick off the wall, we need to drive around the ball and currently we bump it sometimes as we drive while turning, so this ensures that we take that a bit safer. Maybe a better solution later could be to add a parameter for max angular speed to the move primitive

Reply:
Makes sense, could we reuse `DRIBBLE` here and lower angular acceleration for that mode?

Our angular orientation should be slow when we have the ball anyways